tests/integration/gcp.py

```diff
@@ -104,47 +99,27 @@ def _gcp_create_bucket(self, bucket_name):
 
 
     def _gcp_wait_for_operation(self, operation):
-        self.logger.info(f"Waiting for operation {operation['name']} to complete...")
```

Comment:
We should probably leave some log messages here, to better understand what is happening in the background. The following snippet is an good example, maybe we can stick to it:
* https://github.com/googleapis/python-compute/blob/HEAD/samples/snippets/firewall/delete.py